CONTRIBUTING.md

```diff
@@ -1,108 +1,123 @@
-# How to contribute model to Open Model Zoo
+# How to Contribute Models to Open Model Zoo
 
-We appreciate your intention to contribute model to OpenVINO&trade; Open Model Zoo (OMZ). This guide would help you and explain main issues. OMZ is licensed under the Apache License, Version 2.0. By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms. Please note, that we accept models under permissive licenses, as **MIT**, **Apache 2.0**, **BSD-3-Clause**, etc. Otherwise, it may take longer time to get approve (or even refuse) for your model.
+We appreciate your intention to contribute model to the OpenVINO&trade; Open Model Zoo (OMZ). OMZ is licensed under the Apache\* License, Version 2.0. By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms. Note that we accept models under permissive licenses, such as **MIT**, **Apache 2.0**, and **BSD-3-Clause**. Otherwise, it might take longer time to get your model approved.
 
-Nowadays OMZ supports models from frameworks: 
+Frameworks supported by the Open Model Zoo: 
 * Caffe\*
 * Caffe2\* (via conversion to ONNX\*)
 * TensorFlow\*
 * PyTorch\* (via conversion to ONNX\*)
 * MXNet\*
 
-## Pull request requirements
+## Pull Request Requirements
 
-Contribution to OMZ comes down to creating pull request (PR) in this repository. Please use `develop` branch when creating your PR. Pull request is strictly formalized and must contain:
-* configuration file `model.yml` (learn more in [Configuration file](#configuration-file) section)
-* documentation of model in markdown format (learn more in [Documentation](#documentation) section)
-* accuracy validation configuration file (learn more in [Accuracy Validation](#accuracy-validation) section)
+To contribute to OMZ, create a pull request (PR) in this repository using the `develop` branch.
+Pull requests are strictly formalized and are reviewed by the OMZ maintainers for consistence and legal compliance.
+
+Each PR must contain:
```

Comment:
`Each` is very strong condition, because someone want just add demo or fix minor issue. This guide is *only* about model contribution.